flocker/node/agents/_logging.py

```diff
@@ -7,12 +7,8 @@
 
 from eliot import Field, ActionType, MessageType
 
-# Begin: Helper datastructures to log all
-# IBlockDeviceAPI calls from AWS storage driver using Eliot.
-# - Log all IBlockDeviceAPI calls as Eliot ``ActionType``.
-# - Log ``EC2ResponseError`` from AWS to Boto.
-# - Log failure of attached device manifestation
-# - Log OS out of available devices for attaching volume.
+# Begin: Common structures used by all (AWS, OpenStack)
+# storage drivers.
```

Comment:
I'm not a huge fan of this style of comment block.  People usually don't pay close enough attention when they're maintaining the code later on and the comments end up out-of-date with the implementation (ie, wrong).

If there's a bunch of related code and the grouping seems important enough to highlight then it's probably worth moving all of that code into a new module.  New modules aren't quite free but they're most likely cheap enough that it's not worth thinking about their cost.  This also forces you to think about a name to describe the collection and like to believe that thinking about names helps produce better abstractions by forcing you to understand what you're actually doing.